recipe/meta.yaml

```diff
@@ -1,125 +1,87 @@
+{% set name = "pytensor" %}
 
 package:
-  # https://github.com/conda/conda-build/issues/3933
-  name: pytensor-suite
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytensor/pytensor-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: fa82d29ef0594e0a2751eaad40882dcd6af51333c28be0d79c1d0de9f3a81a86
-  patches:
 
 build:
-  number: 3
+  number: 0
+  script:
+    - python -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - pytensor-cache = bin.pytensor_cache:main
 
-outputs:
-  - name: pytensor-base
-    build:
-      script:
-        - python -m pip install . --no-deps -vv
-      entry_points:
-        - pytensor-cache = bin.pytensor_cache:main
-    requirements:
-      build:
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - python                                 # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - cython                                 # [build_platform != target_platform]
-        # Allow numpy pinnings to be managed by conda-forge global dependencies
-        - numpy                                  # [build_platform != target_platform]
-        - versioneer ==0.28                      # [build_platform != target_platform]
-        - tomli                                  # [build_platform != target_platform]
-        - setuptools >=48.0.0                    # [build_platform != target_platform]
-      host:
-        - pip
-        - python
-        - cython
-        # Allow numpy pinnings to be managed by conda-forge global dependencies
-        - numpy
-        - versioneer ==0.28
-        - tomli
-        - setuptools >=48.0.0
-      run:
-        - python
-        # Ref: <https://github.com/pymc-devs/pytensor/blob/rel-{{ version }}/setup.py>
-        # numpy global dependency is greater than 1.17 on conda-forge
-        # so let conda-forge manage it.
-        - {{ pin_compatible("numpy") }}
-        - scipy >=0.14
-        - filelock
-        - etuples
-        - logical-unification
-        - miniKanren
-        - cons
-        - typing_extensions
-        - setuptools >=48.0.0
-    test:
-      imports:
-        - pytensor
-      commands:
-        - pytensor-cache help
-        - pip check
-        - python check-for-warnings.py allowed-warnings-base.txt
-      requires:
-        - pip
-      files:
-        - check-for-warnings.py
-        - allowed-warnings-base.txt
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - pip
+    - numpy {{ numpy }}
+    - versioneer 0.28
+    - tomli
+    - setuptools >=48.0.0
+    # mkl-service is only available for linux-64, win-64, and osx-64.
+    # For everything else, use openblas.
+    - mkl {{ mkl }}            # [blas_impl == "mkl"]
```

Comment:
I am pretty sure you don't need this in `host` or `run`, since this package doesn't build against these deps explicitly. At runtime the right backend would be pulled in by `numpy`.

Reply:
ok, I assumed this was necessary because you left it in the suggested recipe and also it was in the `conda-forge` recipe. Obviously yours is a good enough explanation for not needing it. I will remove it.